diff -urN bioapi-1.2.2.org/addins/dl/mds/dal_classes.h bioapi-1.2.2/addins/dl/mds/dal_classes.h
--- bioapi-1.2.2.org/addins/dl/mds/dal_classes.h	2005-06-10 02:35:44.000000000 +0200
+++ bioapi-1.2.2/addins/dl/mds/dal_classes.h	2006-04-08 21:32:21.236407000 +0200
@@ -455,7 +455,7 @@
 		DAL_DATABASE_INFO_REF_PTR &prefDatabase,
         PORT_MUTEX_HANDLE &Mutex);
 
-	CSSM_RETURN DAL_DATABASE_INFO_LIST::GetDBNamesAndParameters(
+	CSSM_RETURN GetDBNamesAndParameters(
 				CSSM_DL_DB_HANDLE DLDBHandle,
 				DAL_DB_OPEN_PARAM_PTR pParam);
 };
