http://code.google.com/p/bioapi-linux/issues/detail?id=4

--- bioapi-linux/addins/pwbsp/Makefile.in.orig	2012-10-22 08:04:44.000000000 +0200
+++ bioapi-linux/addins/pwbsp/Makefile.in	2012-11-01 12:48:56.593549017 +0100
@@ -113,7 +113,7 @@
 CCDEPMODE = @CCDEPMODE@
 CFLAGS = @CFLAGS@
 CPP = @CPP@
-CPPFLAGS = -I${top_srcdir}/include/ \
+AM_CPPFLAGS = -I${top_srcdir}/include/ \
 		-I${top_srcdir}/imports/cdsa/v2_0/inc/
 
 CXX = @CXX@
@@ -139,7 +139,7 @@
 INSTALL_SCRIPT = @INSTALL_SCRIPT@
 INSTALL_STRIP_PROGRAM = @INSTALL_STRIP_PROGRAM@
 LD = @LD@
-LDFLAGS = -Wl,--version-script=${srcdir}/libpwbsp.ver
+AM_LDFLAGS = -Wl,--version-script=${srcdir}/libpwbsp.ver
 LEX = @LEX@
 LEXLIB = @LEXLIB@
 LEX_OUTPUT_ROOT = @LEX_OUTPUT_ROOT@
--- bioapi-linux/addins/qtpwbsp/Makefile.in.orig	2012-10-22 08:04:44.000000000 +0200
+++ bioapi-linux/addins/qtpwbsp/Makefile.in	2012-11-01 12:49:06.603548597 +0100
@@ -122,7 +122,7 @@
 CCDEPMODE = @CCDEPMODE@
 CFLAGS = @CFLAGS@
 CPP = @CPP@
-CPPFLAGS = -I${top_srcdir}/include \
+AM_CPPFLAGS = -I${top_srcdir}/include \
 		-I${top_srcdir}/imports/cdsa/v2_0/inc \
 		${QT_CXXFLAGS}
 
@@ -149,7 +149,7 @@
 INSTALL_SCRIPT = @INSTALL_SCRIPT@
 INSTALL_STRIP_PROGRAM = @INSTALL_STRIP_PROGRAM@
 LD = @LD@
-LDFLAGS = -Wl,--version-script=${srcdir}/libqtpwbsp.ver \
+AM_LDFLAGS = -Wl,--version-script=${srcdir}/libqtpwbsp.ver \
 		${QT_LIBS}
 
 LEX = @LEX@
